docs/README.md

```diff
@@ -49,13 +48,15 @@ Here follows the list of GitHub Actions topics available in the current document
     - [get-commit-message](#get-commit-message)
     - [git-commit-changes](#git-commit-changes)
     - [git-latest-tag](#git-latest-tag)
+    - [github-check-secrets-source](#github-check-secrets-source)
     - [github-check-upcoming-runs](#github-check-upcoming-runs)
     - [github-deployment-create](#github-deployment-create)
     - [github-deployment-status-update](#github-deployment-status-update)
     - [github-deployments-delete](#github-deployments-delete)
     - [github-download-file](#github-download-file)
     - [github-https-auth](#github-https-auth)
     - [github-list-changes](#github-list-changes)
+    - [github-manage-dependabot-pr-approval](#github-manage-dependabot-pr-approval)
```

Comment:
Any naming suggestion is welcome

Reply:
could not even mention dependabot if the actor could become an input